ui/.gitignore

```diff
@@ -0,0 +1,23 @@
+.DS_Store
+node_modules
+/dist
+
+
+# local env files
+.env.local
+.env.*.local
+
+# Log files
+npm-debug.log*
```

Comment:
Which one of those are you actually using? Could you please remove the others?

Reply:
Removed